Description: Makes the configuration file install correctly (it is no binary).
Origin: other
Forwarded: http://www.draisberghof.de/usb_modeswitch/bb/viewtopic.php?t=173
Author: Didier Raboud <didier@raboud.com>
Last-Update: 2009-07-02

Index: usb-modeswitch/Makefile
===================================================================
--- usb-modeswitch.orig/Makefile	2009-05-29 15:12:25.000000000 +0200
+++ usb-modeswitch/Makefile	2009-05-29 15:13:25.000000000 +0200
@@ -22,7 +22,7 @@
 	mkdir -p $(SBINDIR)
 	install ./usb_modeswitch $(SBINDIR)
 	mkdir -p $(ETCDIR)
-	install ./usb_modeswitch.conf $(ETCDIR)
+	install --mode=644 ./usb_modeswitch.conf $(ETCDIR)
 
 uninstall: clean
 	$(RM) /usr/sbin/usb_modeswitch
